scripts/migrate_user_id_to_integer.py

```
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+user_id 컬럼을 VARCHAR에서 INTEGER로 변경하는 마이그레이션 스크립트
+"""
+import asyncio
+import sys
+import os
+
+# 프로젝트 루트를 Python 경로에 추가
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from app.core.database import get_async_session
```

Pylint (3.3.7)

[error] Unable to import 'app.core.database'

(E0401)